### ISSUE TITLE
original Subtitle Edited when performing actions on Text

### ISSUE THREAD

**Issue by OmrSi** [SubtitleEdit/subtitleedit#2604]
I've noticed this problem recently.
For example, if I do Break/UnBreak on a text I am translating, the same thing happens with original Subtitle.
This didn't use to happen in previous versions.
This should only happen to the text that is being translated, not the original.

On a separate suggestion, when you De-activate The choice to edit original subtitle, you can only view the original in List view, and its text Box disappears, so you cannot see the whole sentence.
Can you maybe let the Text Box for the original subtitle appear but without the ability to Edit it?

**Comment by darnn** [comment]
"Unbreak" does cause both the original and the translation to unbreak, which it did not do in this random beta from February that I opened now. However, "auto break text", which also affects both the original and the translation, has done this in February as well, so there's no change there. But I agree that it really should just affect the textbox that the program's focused on.

**Comment by OmrSi** [comment]
Yeah, when it unbreaks the original it gets errors and is saved, it shouldn't modify the original.

**Comment by OmrSi** [comment]
@niksedk ?

**Comment by niksedk** [comment]
I'm not 100% sure what to do...

**Comment by OmrSi** [comment]
When using auto break/unbreak shourtcut in the textbox it is breaking/unbreaking the original text in addition to the next. 
It didn't use to do that. 
Now the original is being edited.
Maybe prevent it from being edited? Only copy is allowed but no editing?

**Comment by OmrSi** [comment]
I thin; it started happening after allowing the imported time to apply on the Original sub.

**Comment by OmrSi** [comment]
@niksedk

**Comment by OmrSi** [comment]
@niksedk any news?

**Comment by OmrSi** [comment]
@niksedk still don't know why it is happening?
